src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -403,21 +403,44 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
 
     @computed get clinicalAttributeIdToClinicalAttribute():{[clinicalAttributeId:string]:ClinicalAttribute} {
         if (this.props.store.clinicalAttributes.isComplete) {
-            return this.props.store.clinicalAttributes.result.reduce((map:{[clinicalAttributeId:string]:ClinicalAttribute}, next)=>{
+            let _map: {[clinicalAttributeId: string]: ClinicalAttribute} = this.props.store.clinicalAttributes.result.reduce((map:{[clinicalAttributeId:string]:ClinicalAttribute}, next)=>{
                 map[next.clinicalAttributeId] = next;
                 return map;
             }, {});
+            if (this.props.store.studyIds.isComplete) {
+                _map["total_mutations"] = {
```

Comment:
its better to make this an enum value or something (look at the enum `SpecialAttribute`), at least a constant in this file `const TOTAL_MUTATIONS_CLIN_ATTR_ID = "total_mutations"`, then reference that elsewhere below